testing/diff.test.ts

```diff
@@ -0,0 +1,110 @@
+import diff from "./diff.ts";
```

Comment:
Can you rename this file to `diff_test.ts` to stick to [style guide](https://deno.land/style_guide.html)?

Reply:
Oops, sorry. I'll fix it.

Reply:
Actually all of tests file should use `_test.ts` suffix :)